internal/app/handlers/ping_handler_test.go

```diff
@@ -0,0 +1,156 @@
+package handlers
+
+import (
+	"errors"
+	"github.com/northmule/shorturl/internal/app/logger"
+	"github.com/northmule/shorturl/internal/app/services/url"
+	"github.com/northmule/shorturl/internal/app/storage"
+	"github.com/northmule/shorturl/internal/app/storage/models"
+	"github.com/stretchr/testify/mock"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"testing"
+)
+
+type MockPostgresStorageOk struct {
+	mock.Mock
+}
+
+func (m *MockPostgresStorageOk) Add(url models.URL) error {
+	return nil
+}
+func (m *MockPostgresStorageOk) FindByShortURL(shortURL string) (*models.URL, error) {
+	return nil, nil
+}
+func (m *MockPostgresStorageOk) FindByURL(url string) (*models.URL, error) {
+	return nil, nil
+}
+func (m *MockPostgresStorageOk) Ping() error {
+	return nil
+}
+func (m *MockPostgresStorageOk) MultiAdd(url []models.URL) error {
+	return nil
+}
+
+type MockPostgresStorageBad struct {
+	mock.Mock
+}
+
+func (m *MockPostgresStorageBad) Add(url models.URL) error {
+	return nil
+}
+func (m *MockPostgresStorageBad) FindByShortURL(shortURL string) (*models.URL, error) {
+	return nil, nil
+}
+func (m *MockPostgresStorageBad) FindByURL(url string) (*models.URL, error) {
+	return nil, nil
+}
+func (m *MockPostgresStorageBad) Ping() error {
+	args := m.Called()
+	return args.Error(0)
+}
+func (m *MockPostgresStorageBad) MultiAdd(url []models.URL) error {
+	return nil
+}
+
+func TestPingHandler_CheckStorageConnect(t *testing.T) {
+	_ = logger.NewLogger("fatal")
+
+	file, err := os.CreateTemp("/tmp", "TestFileStorage_Add_*.json")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.Remove(file.Name())
+
+	postgresStorage := new(MockPostgresStorageOk)
+	postgresStorage.On("Ping").Return("Ok")
+
+	tests := []struct {
+		name     string
+		storage  url.StorageInterface
+		wantBody string
+		wantCode int
+	}{
+		{
+			name:     "MemoryStorage",
+			storage:  storage.NewMemoryStorage(),
+			wantBody: "Ok",
+			wantCode: http.StatusOK,
+		},
+		{
+			name:     "MemoryStorage",
```

Comment:
Судя по всему имелся ввиду File?
```suggestion
			name:     " FileStorage",
```